extensions/awork/src/projects.tsx

```diff
@@ -0,0 +1,95 @@
+import { Action, ActionPanel, Icon, launchCommand, LaunchType, List, LocalStorage } from "@raycast/api";
+import { showFailureToast, useCachedPromise } from "@raycast/utils";
+import { useState } from "react";
+import { getProjects, project } from "./composables/FetchData";
+import { authorizationInProgress } from "./composables/WebClient";
+
+const Actions = (props: { projectID: string; isBillable: boolean }) => {
+  const { data: BaseUrl } = useCachedPromise(() => LocalStorage.getItem<string>("URL"));
+
+  return (
+    <ActionPanel>
+      <Action.OpenInBrowser url={`${BaseUrl}/projects/${props.projectID}`} />
+      <Action.CopyToClipboard content={`${BaseUrl}/projects/${props.projectID}`} />
```

Comment:
I though I was copying title but it was URL so let's clarify.
```suggestion
      <Action.CopyToClipboard title="Copy URL to Clipboard" content={`${BaseUrl}/projects/${props.projectID}`} />
```